oms-all-deploy/nestedtemplates/omsAutomation.json

```diff
@@ -0,0 +1,433 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "omsAutomationAccountName": {
+      "type": "string",
+      "defaultValue": "OMSAutomation",
+      "metadata": {
+        "description": "Assign a name for the Automation account"
+      }
+    },
+    "omsAutomationRegion": {
+      "type": "string",
+      "defaultValue": "West Europe",
+      "allowedValues": [
+        "Japan East",
+        "East US 2",
+        "West Europe",
+        "Southeast Asia",
+        "South Central US",
+        "North Europe",
+        "Canada Central",
+        "Australia Southeast",
+        "Central India",
+        "Japan East"
+      ],
+      "metadata": {
+        "description": "Specify the region for your Automation account"
+      }
+    },
+    "omsWorkspaceName": {
+      "type": "string",
+      "defaultValue": "OMSWorkspace",
+      "metadata": {
+        "description": "Assign a name for the OMS workspace"
+      }
+    },
+    "omsRecoveryVaultName": {
+      "type": "string",
+      "defaultValue": "OMSRecovery",
+      "metadata": {
+        "description": "Assign a name for the ASR Recovery Vault"
+      }
+    },
+    "azureAdmin": {
+      "type": "string",
+      "defaultValue": "azureadmin",
+      "metadata": {
+        "description": "Enter your service admin user"
+      }
+    },
+    "azureAdminPwd": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Enter the pwd for the service admin user. The pwd is enrypted during runtime and in the Automation assets"
+      }
+    },
+    "_artifactsLocation": {
+      "type": "string",
+      "defaultValue": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/oms-all-deploy",
```

Comment:
Since this is a nested template I think you want to remove the default value to ensure this is passed in an matches the global setting for artifacts